client/src/components/Pages/login/index.js

```diff
@@ -0,0 +1,102 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+import {
+  NotificationContainer,
+  NotificationManager,
+} from 'react-notifications';
+
+import loginSchema from '../../utils/validationSchemas/loginSchema';
+import LabeldInput from '../../common/LabeledInput';
+import Button from '../../common/Button';
+import 'react-notifications/lib/notifications.css';
+import './style.css';
+
+class Login extends Component {
+  state = {
+    account: {
+      email: '',
+      password: '',
+    },
+    error: {},
+  };
+
+  handleSubmit = async event => {
+    event.preventDefault();
+    const { account: loginAccount } = this.state;
+    try {
+      await loginSchema.validate(loginAccount, {
+        abortEarly: false,
+      });
+      await axios.post('/api/v1/login', loginAccount);
+    } catch (err) {
+      if (err.name === 'ValidationError') {
+        const errorObj = {};
+        err.inner.forEach(fieldError => {
+          errorObj[fieldError.path] = fieldError.message;
+        });
+        this.setState({ error: errorObj });
+      } else {
+        NotificationManager.error(err.response.data.message);
+      }
+    }
+  };
+
+  handleChange = ({ currentTarget: input }) => {
+    const { account: loginAccount } = this.state;
+    const account = { ...loginAccount };
+    account[input.name] = input.value;
+    this.setState({ account });
+  };
+
+  render() {
+    const { account, error } = this.state;
+    return (
+      <form onSubmit={this.handleSubmit} action="">
+        <NotificationContainer />
+        <div className="loginContainer">
+          <div className="loginContainer_input">
+            <h1 className="loginContainer__head">Log In</h1>
+            <LabeldInput
+              autoFocus
+              htmlFor="email"
+              label="Email"
+              id="1"
+              type="text"
+              placeHolder="ex.fadi"
```

Comment:
Is this a correct placeholder for email?
```suggestion
              placeHolder="Ex. fadi@gmail.com"
```